eli5/formatters/html.py

```diff
@@ -40,8 +41,10 @@ def format_as_html(explanation, include_styles=True, force_weights=True,
     in the document.
     If ``highlight_spaces`` is None (default), spaces will be highlighted in
     feature names only if there are any spaces at the start or at the end of the
-    feature. Setting it to True forces space highlighting, and setting it to False
-    turns it off.
+    feature. Setting it to True forces space highlighting, and setting it to
+    False turns it off.
+    If ``horizontal_multitarget`` is True (default), multitarget classifier
```

Comment:
the option is named "horizontal_layout". I'd also use "multiclass" term instead of "multitarget" because it is more common for classifiers.